fendermint/vm/interpreter/src/fvm/mod.rs

```diff
@@ -27,7 +28,7 @@ pub use query::FvmQueryRet;
 use tendermint_rpc::Client;
 
 pub use self::broadcast::Broadcaster;
-use self::state::ipc::GatewayCaller;
+use self::{state::ipc::GatewayCaller, upgrade_scheduler::UpgradeScheduler};
```

Comment:
```suggestion
use self::{state::ipc::GatewayCaller, upgrades::UpgradeScheduler};
```

Maybe we can simplify the module name?